index.js

```diff
@@ -17,6 +18,17 @@ const httpsAgent = new https.Agent({
   maxCachedSessions: 1,
   keepAlive: true
 });
+httpsAgent.createConnection = function(options, callback) {
```

Comment:
Reading the Node source, this doesn't seem to be a solid replacement, at minimum parameter-wise, for createConnection: https://github.com/nodejs/node/blob/98e54b0bd4854bdb3e2949d1b6b20d6777fb7cde/lib/https.js#L75